src/Scenario/ClientGenerator.php

```diff
@@ -22,13 +31,18 @@ public function getStatusWatcherClient(StatusWatcherCallbacks $callbacks)
         return new StatusWatcherClient($callbacks);
     }
 
-    public function getAuthKeyInfo(): string
+    /**
+     * @throws TGException
+     *
+     * @return string
+     */
+    public function getAuthKey(): string
     {
-        return trim(file_get_contents('./first.authkey'));
-    }
+        $envPath = getenv($this->envName);
+        if (!$envPath || !file_exists($envPath)) {
+            throw new TGException(0, "Please set {$this->envName} env var to valid filename");
+        }
```

Comment:
Better to keep in ENV-var authkey itself, no file pathes

Reply:
Let's leave env var for now, to be fixed in future refactoring.